### ISSUE TITLE
ibazel_notify_changes ?

### ISSUE THREAD

**Issue by schmitch** [meetup/rebazel#3]
Is there a way to configure 'ibazel_notify_changes'?
Currently ibazel can be configured that the Binary is only notified for File Changes and does not restart on it's own. I'm currently looking in creating a small launcher for [Playframework](https://www.playframework.com/) and I try to actually only rebuild my bazel target if a file changes and just try to swap the Classpath, as Play does with `sbt`.

**Comment by softprops** [comment]
Hi @schmitch. I'm not sure I understand. Rebazel will by default watch for changes to the input files a target depends on and retriggers an action when they change. What would you like to see in addition to that?

**Comment by schmitch** [comment]
i.e. if I call
`ibazel run //:mytarget` and the target has a label called `ibazel_notify_changes`
than ibazel will rebuild the target but not restart the target i.e. consider a java programm:

```java
public class HelloWorld 
{
       public static void main (String[] args)
       {
             System.out.println("Hello World!");
       }
}
```

it will only print "Hello World!" and if I edit it to "Hello NAME!" it won't print "Hello NAME!" instead it will actually just rebuild the java source file to a new jar and than it will send "BUILD COMPLETED" to the stdin of the current program.

So basically a Java Program could listen via `new BufferedReader(new InputStreamReader(System.in))` to new input and if a input comes in it could just reload the ClassLoader, so it could actually Hot Reload a Webserver.

**Comment by softprops** [comment]
Oh I see. Thanks for clarifying
